app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  def updates_comment_counter
+    post.increment!(:comments_counter)
+  end
+end
+
+# Comment.create(post_id: 1, author_id: 2, text: 'Hi Tom!' )
+# Comment.create(post_id: 1, author_id: 2, text: 'Hello my guy!' )
+# Comment.create(post_id: 1, author_id: 2, text: 'wasup odogu' )
+# Comment.create(post_id: 1, author_id: 2, text: 'Hey hey hwz it' )
+# Comment.create(post_id: 1, author_id: 2, text: 'Long time wasup!' )
+# Comment.create(post_id: 1, author_id: 2, text: 'URISEY IWE' )
```

Comment:
- [OPTIONAL] Please kindly remove commented code from your code base as this makes your code base dirty and less readable.

Reply:
Thank you for noting this @t-yanick